src/attributes/classes.js

```diff
@@ -78,7 +76,7 @@ jQuery.fn.extend({
 
 				if ( cur ) {
 					j = 0;
-					while ( (clazz = classes[j++]) ) {
+					while ( ( clazz = classes[ j++ ] ) ) {
```

Comment:
What @dmethvin said. It _was_ more readable (in my opinion), but then we started adding padding spaces to expression parentheses. I'm hoping jscs-dev/node-jscs/pull/1254 will gain some traction so we can change back.